testsuite/features/finishing/srv_count_scc_access.feature

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@srv_scc_access_logging
+Feature: Test for excessive SCC accesses
+
+  Scenario: Count UI SCC accesses in rhn_web_ui.log
+    Then the word "scc.suse.com" does not occur more than 100 times in "/var/log/rhn/rhn_web_ui.log" on "server"
+
+  Scenario: Count Taskomatic SCC accesses in rhn_taskomatic_daemon.log
+    Then the word "scc.suse.com" does not occur more than 50 times in "/var/log/rhn/rhn_taskomatic_daemon.log" on "server"
```

Comment:
```suggestion
    Then the word "scc.suse.com" does not occur more than 50 times in "/var/log/rhn/rhn_taskomatic_daemon.log" on "server"

```
A newline at the end is missing. The rest is simply a port from 4.3 I guess?

Reply:
Yup, just a direct port. The only thing that _might_ require changing are the threshold values, but sadly I don't have enough time left in this team, as today is very definitely my last day in SUMA QE.